lib/web_ui/dev/steps/compile_tests_step.dart

```diff
@@ -110,7 +110,8 @@ Future<void> copySkiaTestImages() async {
     'images',
   ));
 
-  for (final io.File imageFile in testImagesDir.listSync(recursive: true).whereType<io.File>()) {
+  for (final io.File imageFile
+      in testImagesDir.listSync(recursive: true).whereType<io.File>()) {
```

Comment:
nit: please undo formatting-only changes. [Flutter style guide does not use](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#overview) `dartfmt`.